js/gated-content/src/mixins/EventMixin.js

```diff
@@ -36,22 +34,28 @@ export const EventMixin = {
       return this.$store.getters.getAppSettings;
     },
     date() {
-      return moment(this.video.attributes.date.value).format('dddd, MMMM Do, YYYY');
+      return dayjs(new Date(this.video.attributes.date.value)).format('dddd, MMMM Do, YYYY');
     },
     time() {
-      return moment(this.video.attributes.date.value).format('h:mm a');
+      return dayjs(new Date(this.video.attributes.date.value)).format('h:mm a');
     },
     duration() {
-      return moment.duration(moment(this.video.attributes.date.value)
-        .diff(moment(this.video.attributes.date.end_value))).humanize();
+      return `${Math.floor(dayjs.duration(
+        new Date(this.video.attributes.date.end_value) - new Date(this.video.attributes.date.value),
+      ).asMinutes())} minutes`;
     },
     startsIn() {
-      const duration = moment.duration(moment(this.video.attributes.date.value)
-        .diff(moment()));
-      if (duration.asHours() > 48) {
-        return duration.format('d [day]');
+      const eventStartDate = new Date(this.video.attributes.date.value);
+      const startsDuration = dayjs.duration(eventStartDate - new Date());
+
+      if (startsDuration.asHours() >= 48) {
+        return `${Math.floor(startsDuration.asDays())} days`;
```

Comment:
Instead of such string, can we use something like:
```suggestion
        return startsDuration.format('d');
```
?

Reply:
startsDuration.format('d') returns only the number